manifests/l/LibreWolf/LibreWolf/103.0.1-1/LibreWolf.LibreWolf.installer.yaml

```diff
@@ -0,0 +1,29 @@
+yaml-language-server: =https://aka.ms/winget-manifest.installer.1.1.0.schema.json
```

Comment:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
```

Reply:
cc @maltejur: You might want to take a look at why this is being messed up?

Reply:
Yeah something went wrong there, I'll fix it.